src/Common/src/TypeSystem/Common/TypeSystemHelpers.cs

```diff
@@ -266,11 +266,16 @@ public static MethodDesc FindVirtualFunctionTargetMethodOnObjectType(this TypeDe
         }
 
         /// <summary>
-        /// Given Foo&lt;T&gt;, returns Foo&lt;!0&gt;.
+        /// Creates an open instantiation of a type. Given Foo&lt;T&gt;, returns Foo&lt;!0&gt;.
+        /// If the type is not generic, returns the <paramref name="type"/>.
         /// </summary>
-        private static InstantiatedType InstantiateAsOpen(this MetadataType type)
+        public static TypeDesc InstantiateAsOpen(this TypeDesc type)
```

Comment:
I believe this terminology for Open is quite confusing. It really means instantiate such that if the type parameters from an open instantiation were substituted in, the result would be the same as the type definition. Which is really bizarre. (The open instantiation is used to refer to the uninstantiated form in a number of documents, and is the same thing as the typical instantiation.)